options/posix/include/mlibc/posix-sysdeps.hpp

```diff
@@ -178,7 +178,7 @@ int sys_vm_unmap(void *pointer, size_t size);
 	socklen_t *actual_length);
 [[gnu::weak]] int sys_gethostname(char *buffer, size_t bufsize);
 [[gnu::weak]] int sys_sethostname(const char *buffer, size_t bufsize);
-[[gnu::weak]] int sys_mkfifoat(int dirfd, const char *path, int mode);
+[[gnu::weak]] int sys_mkfifoat(int dirfd, const char *path, mode_t mode);
```

Comment:
This should be changed in all sysdeps where it's relevant.

Reply:
Fixed